packages/trader/src/App/Components/Elements/Modals/UnsupportedContractModal/unsupported-contract-modal.jsx

```diff
@@ -21,10 +21,7 @@ class UnsupportedContractModal extends React.Component {
                 is_closed_on_cancel
                 is_visible={is_visible}
             >
-                <Localize
-                    i18n_default_text='This trade type is currently not supported on {{website_name}}. Please go to Binary.com for details.'
-                    values={{ website_name }}
-                />
+                <Localize i18n_default_text='You’ve selected a trade type that is currently unsupported but we’re working on it.' />
```

Comment:
Shouldn't there be a comma before `but`?

Reply:
this is as per copywriters